js/ui/popover/ui.popover.js

```diff
@@ -258,6 +258,8 @@ const Popover = Popup.inherit({
 
         this.$element().addClass(POPOVER_CLASS);
         this.$wrapper().addClass(POPOVER_WRAPPER_CLASS);
+
+        this.setAria('role', this.option('toolbarItems')?.length ? 'dialog' : 'tooltip');
```

Comment:
It improves readability and simplifies a future debug.
```suggestion
        const isInteractable = this.option('toolbarItems')?.length;
        this.setAria('role', isInteractable  ? 'dialog' : 'tooltip');
```